javascript/index.js

```diff
@@ -15,3 +15,128 @@ mobileMenuItems.forEach((item) => {
     document.querySelector('.mobile-menu').classList.remove('show-menu');
   });
 });
+
+// Popup window
+
+const portfolio = document.querySelector('#portfolio');
+
+const projectArray = [
+  {
+    mobileImg: 'images/tonic-section.svg',
+    desktopImg: 'images/desktop-snap3.png',
+    cardTitle: 'Tonic',
+    cardDesc: 'A daily selection of privately personalized reads; no accounts or sign-ups required',
+    class: 'card',
```

Comment:
- [ ] Great job so far. your list of technologies and your `see live and see source` should be coming from your array- objects as well. this helps to make each of them unique to a particular project. kindly fix that.